packages/stream/get-stream/get-stream.test.js

```diff
@@ -0,0 +1,132 @@
+          'Failed to get the stream data because the stream ID is empty. Please provide a stream ID'
+        )
+      }
+    })
+
+    it('should return error response if function is called with stream ID with not a number type', async function () {
+      try {
+        await getStream('1')
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to get the stream data because the stream ID is not in number format. A stream ID must be number format'
+        )
+      }
+
+      try {
+        await getStream({})
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to get the stream data because the stream ID is not in number format. A stream ID must be number format'
+        )
+      }
+
+      try {
+        await getStream([])
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('TypeError')
+        expect(error.message).to.be.equal(
+          'Failed to get the stream data because the stream ID is not in number format. A stream ID must be number format'
+        )
+      }
+    })
+
+    it('should return error response if stream ID not found', async function () {
+      let stream_id = 1000
+      nock(`${Internal.config.api.base_url}`)
+        .get(`/${Internal.config.api.version}/streams/${stream_id}`)
+        .reply(404, { code: 404, message: 'ID not found', data: '' })
+
+      try {
+        await getStream(stream_id)
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('Error')
+        expect(error.message).to.be.equal(
+          'Failed to get the stream data because the stream ID is not found. Please provide a valid stream ID.'
+        )
+      }
+    })
+  })
+
+  describe('Positive test', function () {
+    let stream_id = 351
+    beforeEach(function () {
+      nock(`${Internal.config.api.base_url}`)
+        .get(`/${Internal.config.api.version}/streams/${stream_id}`)
+        .reply(200, {
+          status: {
+            code: 200,
+            message: 'Success',
+            type: 'success',
+          },
+          data: {
+            id: stream_id,
+            name: 'a new stream',
+            slug: '',
+            hls_manifest_path: '',
+            dash_manifest_path: '',
+            description: '',
+            created_by: 6,
+            created_at: '2022-09-05T07:38:09.525329768Z',
+            updated_at: '2022-09-05T07:38:09.525329768Z',
+            quality: '360',
+            viewer_count: 0,
+          },
+        })
+    })
+
+    afterEach(function () {
+      return nock.cleanAll()
+    })
+
+    it('should call the getStream function', async function () {
+      const result = await getStream(stream_id)
+      // console.log(result)
```

Comment:
Remove this console.log code here

Reply:
@faiq-naufal done removing unused console log at commit [c5a4c44](https://github.com/inlivedev/inlive-js-sdk/pull/12/commits/c5a4c44fd9460b32b50af9bfcfc68d2552fd3782)